package/yast2-schema.changes

```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Fri Dec 28 15:02:56 CET 2018 - schubi@suse.de
+
+- Added initial yast2-tftp-server (bsc1108199).
```

Comment:
initial sounds strange here. What about Added newly added schema for yast2-tftp-server ?